src/helper/network/acceptor.cpp

```diff
@@ -28,15 +28,16 @@ acceptor::acceptor(const std::string_view &sv)
         throw std::system_error(errno, std::generic_category());
     }
 
-    struct sockaddr_un addr{};
+    struct sockaddr_un addr {
+    };
```

Comment:
There might be an option for single-line statements to avoid this type of formatting.